src/commands/kick.ts

```diff
@@ -0,0 +1,31 @@
+import Command from "../command";
+import * as DiscordJS from "discord.js";
+import * as VoteSystem from "../vote-system";
+
+export = new Command
+(
+	"kick",
+
+	async (message: DiscordJS.Message) =>
+	{
+		if (VoteSystem.Poll.currentPoll === null)
+		{
+			VoteSystem.Poll.currentPoll = VoteSystem.Poll.standardPoll(message, "kick", (member: DiscordJS.GuildMember) => member.kick("You have been voted off the island"), (2 / 3));
```

Comment:
I'm not sure where the string passed to `member.kick` goes. If you can, please try to figure out what happens to it. I have a feeling that it gets put in the audit log.

Reply:
yup, that's the only place it is found

Reply:
I think "This person was voted to be kicked" will be better because it is talking to the person reading the audit log.

Reply:
I was going to suggest a similar change. Go for it